be/src/exec/spill/spiller.cpp

```diff
@@ -133,59 +59,46 @@ Status Spiller::prepare(RuntimeState* state) {
         }
     }
 
-    _file_group = std::make_shared<SpilledFileGroup>(*_spill_fmt);
+    ASSIGN_OR_RETURN(_serde, spill::create_serde(&_opts));
+    _block_group = std::make_shared<spill::BlockGroup>(_serde);
+#ifndef BE_TEST
+    _block_manager = state->query_ctx()->spill_block_manager();
```

Comment:
what's the lifecycle of `_block_manager `? `_block_manager ` is per query or long live?

Reply:
> what's the lifecycle of `_block_manager `? `_block_manager ` is per query or long live?

_block_manager is maintained inside each query, not long live